### ISSUE TITLE
How to use example build in arm64 mac?

### ISSUE THREAD

**Issue by Tanoshi0625** [streamlabs/obs-studio-node#1393]
My environment:
nodejs: 18.12.0
electron: 28.1.1
macOS: 14.1.1 (23B81)

How to use Example build script?  I typed `cmake .. -G"Visual Studio 17 2022" -A x64 -DCMAKE_PREFIX_PATH=%CD%/libobs-src/cmake/`,  but i failed.

Would you help me? Please tell me how to use it.

**Comment by summeroff** [comment]
We have an issue with building recent version of libobs for macOS. So right now osn only can be build for windows too.

There is a latest version before this changes https://github.com/stream-labs/obs-studio-node/releases/tag/0.23.107

as to building steps. above is from steps for windows. 
i do not have exact commands around but you can check how script building it on server. 
https://github.com/stream-labs/obs-studio-node/blob/15c6b06136d427d48a6b53a58ec5b317009e10ab/.github/workflows/main.yml#L78

in short it will be : set env variables and then run scripts to configure and build. 
./ci/install-dependencies-osx.sh
./ci/configure-osn-osx.sh
 ./ci/build-osn-osx.sh

**Comment by Tanoshi0625** [comment]
> We have an issue with building recent version of libobs for macOS. So right now osn only can be build for windows too.
> 
> There is a latest version before this changes https://github.com/stream-labs/obs-studio-node/releases/tag/0.23.107
> 
> as to building steps. above is from steps for windows. i do not have exact commands around but you can check how script building it on server.
> 
> https://github.com/stream-labs/obs-studio-node/blob/15c6b06136d427d48a6b53a58ec5b317009e10ab/.github/workflows/main.yml#L78
> 
> in short it will be : set env variables and then run scripts to configure and build. ./ci/install-dependencies-osx.sh ./ci/configure-osn-osx.sh ./ci/build-osn-osx.sh

alright, thank you for answering my question
